var/spack/repos/builtin/packages/py-pynn/package.py

```diff
@@ -15,18 +15,22 @@ class PyPynn(PythonPackage):
     pypi = "PyNN/PyNN-0.8.3.tar.gz"
     git      = "https://github.com/NeuralEnsemble/PyNN.git"
 
+    version('0.9.5', sha256='91af2126b639a6a795bfc2709ac49423278c4794b6d0da143908b9afcb415f80')
     version('0.9.1', sha256='bbc60fea3235427191feb2daa0e2fa07eb1c3946104c068ac8a2a0501263b0b1')
     version('0.8.3', sha256='9d59e6cffa4714f0c892ec6b32d1f5f8f75ba3a20d8635bac50c047aa6f2537e')
     version('0.8beta', commit='ffb0cb1661f2b0f2778db8f71865978fe7a7a6a4')
     version('0.8.1', sha256='ce94246284588414d1570c1d5d697805f781384e771816727c830b01ee30fe39')
     version('0.7.5', sha256='15f75f422f3b71c6129ecef23f29d8baeb3ed6502e7a321b8a2596c78ef7e03c')
 
-    depends_on('python@2.6:2.8,3.3:')
+    depends_on('python@2.6:2.8,3.3:',   type=('build', 'run'))
     depends_on('py-jinja2@2.7:',        type=('build', 'run'))
     depends_on('py-docutils@0.10:',     type=('build', 'run'))
```

Comment:
I don't think jinja2 or docutils are required anymore, right?

Reply:
It is still required for the documentation build, whether that is actually done right now, I do not know, but I'd prefer to keep it around

Reply:
Can you check whether or not the documentation is built? I don't even see this dependency listed in older versions. Most of the time, the documentation is only built if someone explicitly runs:
```console
$ cd doc
$ make
```
which our Spack packages don't do.

Reply:
Ping @healther